recipes/quip/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: quip
+  version: 1.0  
```

Comment:
please match the package version with the version of the software.

Reply:
To prevent this problem in future use the following style example:

```
{% set name = "quip" %}
{% set version = "1.1" %}
{% set hash = "ce55cb605b792c601577e9aaea3b84d4813e4f32e6661827858e9dce936eb963" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

build:
  number: 1

source:
  url: http://SOMETHING-{{ version }}.tar.gz
  sha256: {{ hash }}
```